### DISCUSSION TITLE
Cannot send solana devnet transaction through chrome extension - network fee not showing

### DISCUSSION THREAD

**Discussion by morikab** [phantom/docs#195] [Help & Support] [answered]
### Summary

![image](https://github.com/phantom/docs/assets/30197840/46123a30-ed54-4693-93e2-012b58b7aada)

Missing network fees disables the send button for a solana devnet transaction. 

### Steps to Reproduce

Trying to initiate SPL transaction.

### Phantom Version

24.3.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @morikab,

This was likely a temporary issue due to degraded devnet performance. Please let me know if the issue persists for you. I was unable to reproduce.

**Reply by morikab** [reply]
The issue still persists for a couple of days now.

**Reply by adamdelphantom** [reply]
@morikab If this is still persistent please provide the phantom version and os, as well as the token address of the token you are attempting to send.
